core/templates/catalog/checked-out.html

```diff
@@ -3,19 +3,19 @@
 <script type="text/javascript" src="/js/jquery.tablesorter.min.js"></script> 
 <form id="live-search" action="" class="styled" method="">
     <fieldset>
-        <input type="text" class="text-input" id="filter" value="" />
+        <input type="text" class="text-input" id="filter" value="" placeholder="Search"/>
         <span id="filter-count"></span>
     </fieldset>
 </form>
 <table id=checked-out-items class="checkIN tablesorter">
   <thead>
     <tr>
-      <td><input type=checkbox></td>
+      <th id="checkBox"></th>
```

Comment:
Can you put this checkbox back in? Or maybe replace with a button that says "Check All" or something? The purpose of this was to allow the lab attendants to select all of the items at once and perform a batch operation on them.